app/lib/fake/server/fake_default_service.dart

```diff
@@ -51,6 +51,9 @@ class FakePubServer {
 
           final server = await IOServer.bind('localhost', port);
           serveRequests(server.server, (request) async {
+            if (request.requestedUri.path.contains('//')) {
+              return shelf.Response.badRequest();
+            }
```

Comment:
Let's not deviate from what the real server does, not when we don't have to... this could cause issues elsewhere.

In general an extra slash..